pylint/checkers/variables.py

```diff
@@ -895,6 +897,8 @@ def _is_variable_violation(node, name, defnode, stmt, defstmt,
             # defined in global or builtin scope
             if defframe.root().lookup(name)[1]:
                 maybee0601 = False
+                use_outer_definition = stmt == defstmt and \
```

Comment:
Let's not use ``\`` for line continuation, instead, let's use parentheses, as in:

```
(
    stmt == defstmt
    and ...
)
```